src/System.Drawing.Common/tests/DrawingTest.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information.
+
+using System.Drawing.Imaging;
+using System.IO;
+using Xunit;
+
+namespace System.Drawing.Tests
+{
+    public abstract class DrawingTest
```

Comment:
Should we name this DrawingTestBase?

Reply:
I haven't used Base in other assemblies.